projects/plugins/jetpack/extensions/blocks/dialogue/save.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { BASE_CLASS_NAME } from './utils';
+
+/**
+ * Convert a time code string into seconds.
+ *
+ * @param {string} timeCode - like `01:10:59`.
+ * @returns {number} Number of seconds.
+ */
+function convertTimeCodeToSeconds( timeCode ) {
+	const timeParts = timeCode.split( ':' );
+
+	return timeParts
+		.reverse()
+		.reduce(
+			( seconds, timePart, index ) => seconds + Math.pow( 60, index ) * parseInt( timePart ),
+			0
+		);
+}
+
+export default function save( { attributes } ) {
+	const { content, participantLabel, showTimestamp, timestamp } = attributes;
+
+	return (
+		<div>
+			<div className={ `${ BASE_CLASS_NAME }__meta` }>
+				<div className={ `${ BASE_CLASS_NAME }__participant` }>
+					{ participantLabel }
+				</div>
+				{ showTimestamp && (
+					<div className={ `${ BASE_CLASS_NAME }__timestamp` }>
+						<a
```

Comment:
Should we remove this link if there's no podcast player?
<img width="184" alt="Screen Shot 2021-02-02 at 5 13 14 PM" src="https://user-images.githubusercontent.com/1689238/106669863-4dc36a80-657a-11eb-95c2-e2cbde40f84d.png">

Reply:
Yes! Though I'd like to handle that in a follow-up issue, as this PR is already pretty big, and the functionality here is the same as with the PHP render function.

Reply:
show/hide the timestamps doesn't depend on the podcast player but the conversation/dialogue settings. So, I think, if the user wants to show them... maybe we should let them do it.